backend/app/services/channels/handler.py

```
@@ -1301,65 +1301,170 @@ async def _execute_private_im_create_task(
+                db=db,
+                user=user,
+                team=team,
+                project_id=project_id,
+                message=content,
+            )
+        except HTTPException as exc:
+            detail = exc.detail if isinstance(exc.detail, str) else "本地任务创建失败"
+            await self.send_text_reply(message_context, f"创建本地任务失败：{detail}")
+            return
+        except ValidationError as exc:
+            await self.send_text_reply(message_context, f"创建本地任务失败：{exc}")
```

Ruff (0.15.18)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)